src/operator/nn/pooling.cc

```diff
@@ -274,12 +274,12 @@ void PoolingComputeExCPU(const nnvm::NodeAttrs &attrs, const OpContext &ctx,
 
   // Pooling does not currently support working with views
   if (inputs[0].IsView() || outputs[0].IsView()) {
+    std::cout << "Fall back to Pooling backward pass..." << std::endl;
```

Comment:
Is this intended? We didn't have any log message for fallback execution.

Reply:
Sorry. Forgot to remove this line. Will fix.